src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -542,34 +546,41 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
         width,
         height,
         data: weatherValues,
+        textTestId: 'temp-rh-tooltip-text',
+        bgdTestId: 'temp-rh-graph-background',
         getInnerText: ([key, value]) => {
-          if (key === 'date') {
+          if (key === 'date' && typeof value === 'object') {
```

Comment:
what's the purpose of adding in `&& typeof value === 'object'` ?

Reply:
nvm. I read 3 lines down and then it made sense

Reply:
I am making sure that the value is the Date object.